src/layouts/SingleColumn.scss

```diff
@@ -10,4 +10,9 @@
     row-gap: 60px;
 
     align-items: center;
+
+    .advanced {
+        position: relative;
+        top: -335px;
```

Comment:
This was my attempt at creating the 'pop' effect so that the later "invited created by x client" and the matrix protocol explanation weren't covered. This will be removed when the eh.. 'pop' is removed.

Reply:
Fixed this